src/main/kotlin/com/zepben/evolve/streaming/get/NetworkConsumerClient.kt

```diff
@@ -679,6 +704,10 @@ class NetworkConsumerClient(
 
 }
 
-inline fun <reified T : EquipmentContainer> NetworkConsumerClient.getEquipmentContainer(mRID: String): GrpcResult<MultiObjectResult> {
-    return getEquipmentContainer(mRID, T::class.java)
+inline fun <reified T : EquipmentContainer> NetworkConsumerClient.getEquipmentContainer(
+    mRID: String,
+    includeEnergizingContainers: IncludedEnergizingContainers = IncludedEnergizingContainers.EXCLUDE_ENERGIZING_CONTAINERS,
+    includeEnergizedContainers: IncludedEnergizedContainers = IncludedEnergizedContainers.EXCLUDE_ENERGIZED_CONTAINERS
+): GrpcResult<MultiObjectResult> {
+    return getEquipmentContainer(mRID, T::class.java, includeEnergizingContainers, includeEnergizedContainers)
```

Comment:
Inline functions cannot be mocked or covered AFAIK. I added a test to make sure it calls the java interop version.

Reply:
priase: Yes, inline function have issues with coverage. Good job referencing this rather than ignoring the coverage warnings.